v6/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/kentik/community_sdk_golang/v6
```

Comment:
I think we can do apiv5 and apiv6 as separate top-level folders. This would be the least confusing approach.
We can move current api5 impl to its own folder once all endpoints get merged.

Reply:
So we would have following Go import paths:
`github.com/kentik/community_sdk_golang/apiv5/kentikapi` -> kentikapi.NewClient()
`github.com/kentik/community_sdk_golang/apiv6/kentikapi` -> kentikapi.NewClient()

Good idea. So please rename the v6 to apiv6

Reply:
Created a task:  https://clkentik.atlassian.net/browse/KNTK-193 - Move kentikapi-v5 code to apiv5 directory

Reply:
Resolved